server/graphql/document/src/queries/patient.rs

```diff
@@ -99,6 +101,23 @@ impl PatientNode {
 
         Ok(result)
     }
+
+    pub async fn programs(&self, ctx: &Context<'_>) -> Result<Vec<ProgramNode>> {
+        let context = ctx.service_provider().context()?;
```

Comment:
Although I don't think we will be querying for multiple patients including programs/documents I still think loader is more appropriate here. At least a TODO as you've done for encounters on program

Reply:
Yes agree I added the missing todo